dspace-spring-rest/src/main/java/org/dspace/app/rest/converter/CommunityConverter.java

```diff
@@ -32,6 +33,9 @@
     @Autowired
     private CollectionConverter collectionConverter;
 
+    @Autowired
+    private CommunityConverter communityConverter;
```

Comment:
this is not really needed as the current object is THE community converter. 
Please note that It must be a singleton as it is annotated with component